Makefile

```diff
@@ -59,10 +59,13 @@ build-binaries:
 upload-binaries-to-github:
 	${MAKEFILE_PATH}/scripts/upload-binaries-to-github
 
+build-and-upload-k8s-resources:
+	${MAKEFILE_PATH}/scripts/build-and-upload-k8s-resources-yaml
+
 sync-readme-to-dockerhub:
 	${MAKEFILE_PATH}/scripts/sync-readme-to-dockerhub
 
-release: create-build-dir build-binaries upload-binaries-to-github
+release: create-build-dir build-binaries upload-binaries-to-github build-and-upload-k8s-resources
```

Comment:
as discussed offline can we make the release process | create -> build binaries -> build yaml -> upload